generator/resources/ec2_performance_test_matrix.json

```diff
@@ -4,7 +4,6 @@
         "username": "ec2-user",
         "installAgentCommand": "go run ./install/install_agent.go rpm",
         "ami": "cloudwatch-agent-integration-test-al2*",
-        "caCertPath": "/etc/ssl/certs/ca-bundle.crt",
```

Comment:
We are delete `caCertPath` since performance/stress does not require it as a input. Here is all the required input for the [terraform performance ](https://github.com/aws/amazon-cloudwatch-agent-test/blob/main/terraform/performance/variables.tf#L1-L84) and how are [things get generated by the generator](https://github.com/aws/amazon-cloudwatch-agent-test/blob/main/generator/test_case_generator.go#L16-L29)